src/features/place-bid/PlaceBidForm/FormSteps/ApproveZAuction/ApproveZAuction.tsx

```diff
@@ -0,0 +1,42 @@
+import { TextContent, TextContentProps } from '../ui';
+import { Wizard, ConfirmationProps } from '@zero-tech/zui/components/Wizard';
+
+import styles from '../FormSteps.module.scss';
+
+export interface ApproveZAuctionProps {
+	errorText: TextContentProps['errorText'];
+	onClose: ConfirmationProps['onClickSecondaryButton'];
+	onApproveZAuction: ConfirmationProps['onClickPrimaryButton'];
+}
+
+export const ApproveZAuction = ({
+	errorText,
+	onClose,
+	onApproveZAuction,
+}: ApproveZAuctionProps) => {
+	const primaryButtonText: ConfirmationProps['primaryButtonText'] = errorText
+		? 'Retry'
+		: 'Continue';
+
+	const textContent =
+		'Before you can accept a bid, your wallet needs to approve zAuction. \nYou will only need to do this once. This will cost gas.';
+
+	const confirmationMessage = (
+		<TextContent textContent={textContent} errorText={errorText} />
+	);
```

Comment:
nit: probably cleaner to have the `textContent` variable there and instead chucking the string in the `TextContent` prop. Not a big issue though